unime/src-tauri/tests/tests/get_state.rs

```diff
@@ -39,6 +39,34 @@ async fn test_get_state_create_new() {
     .await;
 }
 
+#[tokio::test]
+#[serial_test::serial]
+async fn test_locale_persistency_during_onboarding() {
```

Comment:
The test name is a little bit misleading: "during onboarding"? I think it would be more fitting to give it a name such as `locale_stays_unchanged_on_profile_creation` because that's what the code actually does, right?

Reply:
sure, changed it